go/vt/servenv/servenv.go

```diff
@@ -266,6 +267,7 @@ func FireRunHooks() {
 func RegisterDefaultFlags() {
 	OnParse(func(fs *pflag.FlagSet) {
 		fs.IntVar(&port, "port", port, "port for the server")
+		fs.StringVar(&bindAddress, "bind_address", bindAddress, "Bind address for the server.")
```

Comment:
1. Why is the flag called `bind_address` (or `bind-address`)? Why not just `host`?
2. What happens if it is not specified? That should be included in the flag help text.

Reply:
1. I think it means the same thing, I don't mind changing it to host. I re-used what was originally on the PR.
2. The output from viper includes the default value for this flag (127.0.0.1)

Reply:
Apologies, the default value was indeed missing, I added a text to explain that if the flag is left empty, all addresses will be used.